Sources/Keystone/Keystone.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+//! Project version number for Keystone.
+FOUNDATION_EXPORT double KeystoneVersionNumber;
+
+//! Project version string for Keystone.
+FOUNDATION_EXPORT const unsigned char KeystoneVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <Keystone/PublicHeader.h>
```

Comment:
This is what we will have to use as a bridging header replacement. Bridging headers are not supported in framework targets. You have to use an umbrella header and make the headers you add to it public.